source/containers/document-pii-detection/parsers/image_analysis/general_ocr/imaug/__init__.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from .operators import *
+
+def transform(data, ops=None):
+    """ transform """
+    if ops is None:
+        ops = []
+    for op in ops:
+        data = op(data)
+        if data is None:
+            return None
+    return data
+
+
+def create_operators(op_param_list, global_config=None):
+    """
+    create operators based on the config
+    Args:
+        params(list): a dict list, used to create some operators
+    """
+    assert isinstance(op_param_list, list), ('operator config should be a list')
+    ops = []
+    for operator in op_param_list:
+        assert isinstance(operator,
+                          dict) and len(operator) == 1, "yaml format error"
+        op_name = list(operator)[0]
+        param = {} if operator[op_name] is None else operator[op_name]
+        if global_config is not None:
+            param.update(global_config)
+        op = eval(op_name)(**param)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If you do not have absolute control over the contents of the string passed to the `eval()` function, they might be maliciously used to execute an arbitrary function such as `__import__(&#39;os&#39;).system(&#34;rm -rf $HOME&#34;)`. If the string is expected to only contain literals, consider using the `ast.literal_eval()` method to safely evaluate strings containing Python values from untrusted sources.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-use-strings-to-call-functions-methods)

Reply:
Resolve by replacing the eval function with creating specific Python classes.